include/enclave/inside/ocalls.h

```diff
@@ -4,6 +4,25 @@
 #include <sgx.h>
 #include <sgx_defs.h>
 
+#include <stdio.h>
+
+// For code outside SGX enclaves we use spdlog macros for logging. We provide
+// similar macros for logging (only in debug mode) from inside the enclave. Note
+// that this logging is very costly as it does an ocall to print the message,
+// as a consequence we no-op it at compile time for release builds
```

Comment:
It took me a while to work out that we no-op `ocallLogDebug`, and not the `SPDLOG_` macros here. Do we still call `snprintf` even in release builds? I would have expected:

```
#ifdef FAASM_SGX_DEBUG
#define SPDLOG_DEBUG_SGX(...) 
    size_t __bufferSize = 512;
    ...
#else
#define SPDLOG_DEBUG_SGX(...)
    // No-op
#endif
```